logging/cluster-logging-release-notes.adoc

```diff
@@ -1,25 +1,66 @@
 [id="cluster-logging-release-notes"]
-= Release notes for Red Hat OpenShift Logging 5.2
+= Release notes for Red Hat OpenShift Logging 5.3
 include::modules/common-attributes.adoc[]
 :context: cluster-logging-release-notes-v5x
 
 toc::[]
 
-[id="openshift-logging-supported-versions"]
-== Supported versions
-
-.{product-title} version support for Red Hat OpenShift Logging (RHOL)
-[options="header"]
-|====
-|        |4.7          |4.8          |4.9
-|RHOL 5.0|X            |X            |
-|RHOL 5.1|X            |X            |
-|RHOL 5.2|X            |X            |X
-|====
-
 include::modules/making-open-source-more-inclusive.adoc[leveloffset=+1]
 
+[id="cluster-logging-supported-versions"]
+== Supported Versions
+include::modules/cluster-logging-supported-versions.adoc[leveloffset=+1]
+
 // Release Notes by version
+[id="cluster-logging-release-notes-5-3-0"]
+== OpenShift Logging 5.3.0
+The following advisories are available for OpenShift Logging 5.3.x:
+
+[id="openshift-logging-5-3-0-new-features-and-enhancements"]
+=== New features and enhancements
+* With this update, authorization requirements for Log Forwarding have been relaxed. Outputs may now be configured with SASL, username/password, or TLS.
```

Comment:
"Relaxed" seems weird to me, we are offering more options. Suggestion:

Log forwarding outputs can now be configured with more TLS options: server-only or mutual authentication, using system default or custom certificates. SASL and username/password authentication are also available for some output types.

Reply:
Agree, but this one is already merged, so the change would need to be in a follow up PR 🙂